src/test/testing-library/Heading.test.tsx

```diff
@@ -1,9 +1,11 @@
 import { render } from '@testing-library/react';
+import { t } from 'i18next';
 
 import Heading from '../../features/landing/components/WhyUs/Heading/Heading';
 
-const subtitleText: RegExp = /Unlimited customization/;
-const headingText: string = 'Why we';
+const subtitleText: RegExp = new RegExp(t('why_us.business_subtitle'));
+
+const headingText: string = t('why_us.heading');
```

Comment:
If the wrong key is used, you won't be able to find the element you intended, and the test will fail. I don't see the need to complicate things. Also, since we're using a function from an existing library, it seems to me that the test is redundant

Reply:
If the wrong key is used, you won't be able to find the element you intended, and the test will fail. I don't see the need to complicate things. Also, since we're using a function from an existing library, it seems to me that the test is redundant

Reply:
@coderabbitai